libraries/rib-base/src/main/java/com/badoo/ribs/android/RibActivity.kt

```diff
@@ -35,7 +35,7 @@ abstract class RibActivity : AppCompatActivity() {
         ActivityIntegrationPoint(
             activity = this,
             savedInstanceState = savedInstanceState,
-            rootViewGroup = rootViewGroup
+            rootViewGroup = { rootViewGroup }
```

Comment:
Access to the rootViewGroup should be delayed to the moment after `onCreate()`, so clients can setup content view after `super.onCreate()` call (see `RibTestActivity`).